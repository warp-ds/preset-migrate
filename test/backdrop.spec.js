import { describe, expect, test } from "vitest";
import { setup } from "./_helpers.js";


setup();

describe('backdrop',  () => {
    test('', async ({ uno }) => {
        const test = ['backdrop-test'];
        const { css } = await uno.generate(test);
        expect(css).toMatchSnapshot();
    });
});
