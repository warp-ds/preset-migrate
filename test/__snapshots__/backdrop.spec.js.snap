// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`backdrop >  1`] = `""`;
